index.html

```diff
@@ -125,18 +187,112 @@ <h3>
+      <aside class="example">
+        <p>
+          The following example shows how to use the Javascript API to control
+          the `entityTransform`:
+        </p>
+        <pre class="html">
+          &lt;model src="3d-assets/teapot"&gt;&lt;/model&gt;
+          &lt;script&gt;           
+            const model = document.querySelector('model');
+
+            function update() {
+              requestAnimationFrame(update);
+              model.entityTransform = new DOMMatrix().rotate(0, performance.now()/10, 0);
+            }
+            
+            requestAnimationFrame(update);
+           &lt;/script&gt;           
+         </pre>
+        <p data-cite="infra geometry-1">
+          See {{DOMMatrix}} for more details on the methods available for 
+          modifying the view transform.
+        </p>
+      </aside>
       <h3>
         Supporting multiple formats
       </h3>
+      <aside class="example">
+        <p>
+          The following example shows a list of potential source formats
+          that can be selected from:
+        </p>
+        <pre class="html">
+           &lt;model&gt; 
+              &lt;source src="3d-assets/teapot.usdz" type="model/vnd.pixar.usd"&gt;
+              &lt;source src="3d-assets/teapot.gltf" type="model/gltf-binary"&gt;
+           &lt;/model&gt;
+         </pre>
+         `&lt;model&gt;` source selection follows `&lt;video&gt;` selection,
+         where the top-level `src` attribute takes precedence if present, 
+         followed by the first compatible `&lt;source&gt;` element encountered.
+      </aside>
       <aside class="issue" data-number="21"></aside>
       <h3>
         Providing fallback content for legacy user agents
       </h3>
+      <aside class="example">
+        <p>
+          The following example shows the inclusion of fallback content:
+        </p>
+        <pre class="html">
+           &lt;model&gt; 
+              &lt;source src="3d-assets/teapot"&gt;
+              &lt;img src="images/teapot-fallback-thumbnail.png"&gt; 
+           &lt;/model&gt;
+         </pre>
+      </aside>
       <aside class="issue" data-number="22"></aside>
+      <h3>
+        Providing an image-based light (IBL)
+      </h3>
+      <aside class="example">
+        <p>
+          The following example shows the inclusion of a custom image-based
+          light
+        </p>
+        <pre class="html">
+          &lt;model environmentmap="images/teahouse-interior-1k.hdr"&gt; 
+             &lt;source src="3d-assets/teapot" &gt;
+          &lt;/model&gt;
+         </pre>
+      </aside>
       <h3>
         Making `model` accessible
       </h3>
+      <aside class="example">
+        <p>
+          The following example shows the inclusion of descriptive language
+          characterizing the appearance of the model:
+        </p>
+        <pre class="html">
+          
+          &lt;figure&gt;
+            &lt;model alt="A gray, ceramic teapot with an innovative spout" &gt; 
```

Comment:
I still think we should get rid of `alt=`... the html around it or inside the element provides way better and more accessible content (eg., `<figurecaption>`).